src/table/__tests__/TableBodyWrapper.spec.jsx

```diff
@@ -33,10 +39,17 @@ describe('<TableBodyWrapper />', async () => {
   });
 
   it('should render 2x2 table body without CellRenderer', () => {
-    sandbox.replace(renderer, 'getCellRenderer', () => {});
+    // eslint-disable-next-line react/prop-types
```

Comment:
This entire test should be removed. It was testing the case you removed from `tableBodyWrapper`